pisek/task_jobs/generator/manager.py

```diff
@@ -0,0 +1,220 @@
+# pisek  - Tool for developing tasks for programming competitions.
+#
+# Copyright (c)   2019 - 2022 Václav Volhejn <vaclav.volhejn@gmail.com>
+# Copyright (c)   2019 - 2022 Jiří Beneš <mail@jiribenes.com>
+# Copyright (c)   2020 - 2022 Michal Töpfer <michal.topfer@gmail.com>
+# Copyright (c)   2022        Jiří Kalvoda <jirikalvoda@kam.mff.cuni.cz>
+# Copyright (c)   2023        Daniel Skýpala <daniel@honza.info>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# any later version.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import random
+from typing import Any, Optional
+
+from pisek.env.env import Env
+from pisek.utils.paths import TaskPath
+from pisek.config.config_types import GenType, DataFormat
+from pisek.jobs.jobs import Job, JobManager
+from pisek.task_jobs.task_manager import TaskJobManager
+from pisek.task_jobs.compile import Compile
+from pisek.task_jobs.data.data import InputSmall
+from pisek.task_jobs.tools import IsClean
+from pisek.task_jobs.checker import CheckerJob
+
+from .input_info import InputInfo
+from .base_classes import (
+    GeneratorListInputs,
+    GenerateInput,
+    GeneratorTestDeterminism,
+    GeneratorRespectsSeed,
+)
+from .cms_old import CmsOldListInputs, CmsOldGenerate
+from .opendata_v1 import (
+    OpendataV1ListInputs,
+    OpendataV1Generate,
+    OpendataV1TestDeterminism,
+)
+
+SEED_RANGE = range(0, 16**8)
+
+
+class PrepareGenerator(TaskJobManager):
+    """Manager that generates inputs and test generator."""
```

Comment:
The manager doesn't generate a test generator. Did you mean "test**s** the generator"?